src/integration.test.ts

```diff
@@ -82,6 +82,7 @@ describe("Integration Test", (): void => {
     expect(core.info).lastCalledWith(stdout);
     expect(core.error).lastCalledWith(stderr);
     expect(core.setFailed).not.toHaveBeenCalled();
+    expect(child_process.exec).toHaveBeenCalledTimes(1);
```

Comment:
Nit: I can see the logic of only asserting call counts after all of the called withs have been asserted in order to maximize the helpfulness of the test output, but it also makes it more difficult to trace the correspondence to the production code and correctly update tests when a mock's called with assert is separated from its call count assert. I don't have a strong opinion on which way is objectively preferable overall, so I would just keep consistency with the existing code base for now, meaning co-locate asserts on the same mock. (That seems easier than adopting this new convention throughout the code base after all.)